js/repeater.js

```diff
@@ -211,6 +211,46 @@
 			return markup;
 		},
 
+		disable: function() {
+			var disable = 'disable';
+			var disabled = 'disabled';
+
+			this.$search.search(disable);
+			this.$filters.selectlist(disable);
+			this.$views.find('label').attr(disabled, disabled);
+			this.$pageSize.selectlist(disable);
+			this.$primaryPaging.find('.combobox').combobox(disable);
+			this.$secondaryPaging.attr(disabled, disabled);
+			this.$prevBtn.attr(disabled, disabled);
+			this.$nextBtn.attr(disabled, disabled);
```

Comment:
pretty sure these should be .prop(disabled, true) instead of attr(disabled, disabled)

http://stackoverflow.com/questions/5874652/prop-vs-attr